cfgov/v1/jinja2/v1/includes/molecules/footnotes.html

```diff
@@ -0,0 +1,27 @@
+{# ==========================================================================
+
+    footnotes.render()
+
+    ==========================================================================
+
+    Description:
+
+    Create footnote markup when given:
+
+    footnotes: An array of Footnote objects
+
+    ========================================================================== #}
+
+{% macro render( footnotes ) %}
+  <div class="block m-footnotes">
+      <h2 id="footnotes">Footnotes</h2>
```

Comment:
Is this ID just to make it the whole section linkable?

Reply:
Oh nevermind, I see the `aria-labelledby` later on.

Reply:
It's for the `aria-labeledby` in the next comment.